_guides/getting_started.md

```diff
@@ -126,113 +231,155 @@ best constant = 0.500000
 ```
 
-## Getting predictions
+## Make predictions with Vowpal Wabbit 
 
-We want to make predictions of course, this can be done by supplying the `-p filename` option. Stdout can be used as below:
+You can make predictions in VW by supplying the `-p filename`. For example, using filename `stdout`:
 
 ```sh
 vw house_dataset -p /dev/stdout --quiet
 ```
 
-The output is:
+The output for this command is:
+
 ```
 0.000000
 0.000000 second_house
 1.000000 third_house
 ```
 
-- The 1st output line `0.000000` is for the 1st example which has an empty tag.
-- The 2nd output `0.000000 second_house` is for the 2nd example. You'll notice the tag appears here. This is the primary use of the tag: mapping predictions to the examples they belong to.
-- The 3rd output `1.000000 third_house` is for the 3rd example. Clearly, some learning happened, because the prediction is now `1.000000` while the initial prediction was set to `0.5`.
+The first line, `0.000000` refers to the first example, which has an empty tag.
+The second line, `0.000000 second_house` refers to the second example. Notice that the tag appears here. The primary use of the tag is mapping predictions to the corresponding examples.
+The third output, `1.000000 third_house` refers to the third example. The initial prediction was set to `0.5`, and the prediction is now `1.000000`. This result means that _some_ learning occurred. 
+
+In the last example, the model was built in memory incrementally, as VW analyzed the examples. In other words, you predicted _while you learned_.
 
-Note that in this last example, we predicted _while we learned_. The model was being incrementally built in memory as VW went over the examples.
+It is more common to learn first, then save the model file. Then, you make predictions using that saved model.
 
-Alternatively, and more commonly, we would first learn and save the model into a file. Later we would predict using the saved model.
+**Use:** `-i house.model` to load the initial model to memory. 
 
-You may load a initial model to memory by adding `-i house.model`. You may also want to specify `-t` which stands for "test-only" (do no learning):
+**Add** `-t` to specify _test-only_ (do no learning). For example:
 
 ```sh
 vw -i house.model -t house_dataset -p /dev/stdout --quiet
 ```
 
-Which would output:
+The output for this command is:
+
 ```
 0.000000
 1.000000 second_house
 0.000000 third_house
 ```
 
-Obviously the results are different this time, because in the first prediction example, we learned as we went, and made only one pass over the data, whereas in the 2nd example we first loaded an over-fitted (25 pass) model and used our data-set `house_dataset` with `-t` (testing only mode). In real prediction settings, one should use a different data-set for testing vs training.
+The results are different for this example because the first prediction example made only one pass over the data, in other words, learning as it worked. 
```

Comment:
This words is confusing and wrong.

The example reference is for the invocation of vw right before this one.